src/core/scene/a-scene.js

```diff
@@ -184,11 +184,21 @@ module.exports = registerElement('a-scene', {
     exitVR: {
       value: function () {
         var self = this;
+        exitFullscreen();
```

Comment:
I think it won't work. Just entering fullscreen does not set the state to `vr-mode`